tailscale/ui/styles.css

```
@@ -0,0 +1,308 @@
+[data-iii-ui='tailscale'] .ts-row-actions,
+[data-iii-ui='tailscale'] .ts-row-actions-head {
+  width: 2.5rem;
+  text-align: right;
+}
+
+[data-iii-ui='tailscale'] .ts-visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)